internal/backend/post.go

```diff
@@ -260,38 +261,6 @@ func (d *delphisBackend) GetMentionedEntities(ctx context.Context, entityIDs []s
 	return entities, nil
 }
 
-// Testing function to keep functionality
```

Comment:
This function has been renamed PostIterCollect, and moved from the backend-level to the datastore-level. I found it more logically appropriate, and also it can be useful for datastore internal functions. Backend functions can access it anyways by accessing the Datastore interface.

Reply:
@tim-harper Any thoughts? I think this is reasonable but since you built this do you want to weigh in?